src/lib/utils.sh

```diff
@@ -116,3 +116,46 @@ function send-log() {
 
   return 0
 }
+
+function sort-logs() {
+  set -euo pipefail
+
+  if [[ "$CAUR_TYPE" != 'primary' ]]; then
+    echo 'Only primary node can do this action.'
+    return 0
+  fi
+
+  local CAUR_CACHE="mktemp /tmp/chaotic/logs-XXXXXXXXXX"
```

Comment:
Wasn't this supposed to execute? anyway, after executing it, check if it didn't fail...

```suggestion
  local CAUR_CACHE="$(mktemp '/tmp/chaotic/logs-XXXXXXXXXX')"
  
  if ![[ -d "$CAUR_CACHE" ]]; then
     echo 'It was not possible to create a temporary directory for this action.'
     return 62 # Nico, find the next number, and put it here, I lost count already.
  fi
```

Reply:
Note that we used to already use the "CAUR_CACHE" name, so pick another for this temporary variable to avoid confusion...

Reply:
Checking if it failed shouldn't be necessary because of set -e, am I wrong?